package.json

```diff
@@ -16,7 +16,7 @@
     "coverage": "nyc --all --reporter=html npm test",
     "docs": "rimraf docs && jsdoc -c jsdoc.json",
     "prepack": "npm run build",
-    "prepublishOnly": "npm install && npm test && ./bin-dev-scripts/standard-version-release.sh",
+    "prepublishOnly": "npm install && ./bin-dev-scripts/standard-version-release.sh",
     "test": "npm run build && mocha test",
     "sync-files-to": "npx onchange --verbose --await-write-finish=2000 'src/**/*' -- ./bin-dev-scripts/sync-to.sh "
   },
```

Comment:
why remove npm test? 

Do not commit the branch version to master. reset lock file also before merging.

Reply:
Removed it for testing. Should write test cases for it and will add it back. 

Yes. Sure.